llm/dyn_ext_server.go

```diff
@@ -258,6 +255,20 @@ func (llm *dynExtServer) Predict(ctx context.Context, predict PredictOpts, fn fu
 					break out
 				}
 
+				switch {
+				case strings.TrimSpace(p.Content) == lastToken:
+					tokenRepeat++
+				default:
+					lastToken = strings.TrimSpace(p.Content)
+					tokenRepeat = 0
+				}
+
+				// 30 picked as an arbitrary max token repeat limit, modify as needed
+				if tokenRepeat > 30 {
```

Comment:
This should be tunable maybe reuse `api.RepeatLastN`?

Reply:
Hmm, this is a good suggestion but I'm hesitant to overload the setting with multiple functionalities.